tide_media.module

```diff
@@ -260,10 +260,10 @@ function tide_media_preprocess_image(&$variables) {
  * Implements hook_views_pre_render().
  */
 function tide_media_views_pre_render(ViewExecutable $view) {
-  if ($view->id() == "tide_media_browser" && (($view->current_display == 'media_browser') || ($view->current_display == 'document_browser'))) {
+  if ($view->id() === 'tide_media_browser' && (($view->current_display === 'media_browser') || ($view->current_display === 'document_browser'))) {
     $view->element['#attached']['library'][] = 'tide_media/media_browser';
     foreach ($view->result as $value) {
-      if ($value->_relationship_entities['field_media_file_target_id']) {
+      if (!empty($value->_relationship_entities['field_media_file_target_id'])) {
```

Comment:
Not all media types have `field_media_file` field, eg. image has `field_media_image` instead.